### DISCUSSION TITLE
Topology Viewer-Filters stopped working after upgrading

### DISCUSSION THREAD

**Discussion by Abdalrhmanalfalaha** [netbox-community/netbox#10127] [General]
hello, 

after upgrading from 3.2.0 to 3.3.0 the plugin "Topology Viewer" stopped working with the error: "**<class 'django.core.exceptions.FieldError'>

Cannot resolve keyword '_termination_a_device_id' into field. Choices are: _abs_length, color, created, custom_field_data, id, journal_entries, label, last_updated, length, length_unit, status, tagged_items, tags, tenant, tenant_id, terminations, type

Python version: 3.8.10
NetBox version: 3.3.0"**

How can I fix this?

**Comment by jeremystretch** [comment]
You'll need to open a bug report for the plugin asking that they update it to support NetBox v3.3.
